scripts/aot.spec.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+const shell = require('shelljs');
+const VERSION = require('../package.json').version;
+const colors = require('colors/safe');
+
+const packagePath = `../../dist/clr-angular/clr-angular-${VERSION}.tgz`;
+const installCommand = `npm i ${packagePath} --save`;
+
+// Moving to the ks-app root to work.
+console.log(colors.yellow('Testing Clarity Artifacts for AOT'));
+process.chdir('./src/ks-app');
+
+// Do package installation.
+console.log(colors.yellow(`Installing latest @clr/angular with: ${packagePath}`));
+shell.exec(installCommand);
+shell.exec('npm i');
```

Comment:
`npm ci`? Here and above?

Reply:
I can change to ci but think that I should move the `installCommand` below so it doesn't get clobbered.